plugins/env/db/migrate/20150603211116_change_env_deploy_group_to_scope.rb

```diff
@@ -0,0 +1,11 @@
+class ChangeEnvDeployGroupToScope < ActiveRecord::Migration
+  def change
+    remove_index :environment_variables, name: "environment_variables_unique_deploy_group_id"
+    add_column :environment_variables, :scope_type, :string
+    rename_column :environment_variables, :deploy_group_id, :scope_id
+
+    EnvironmentVariable.update_all scope_type: "DeployGroup"
```

Comment:
only need that if you are using update_attribute etc, but not for
update_column or update_all

On Wed, Jun 3, 2015 at 4:41 PM, Roman Sandler <notifications@github.com>
wrote:

> In
> plugins/env/db/migrate/20150603211116_change_env_deploy_group_to_scope.rb
> <https://github.com/zendesk/samson/pull/438#discussion_r31682019>:
>
> > @@ -0,0 +1,11 @@
> > +class ChangeEnvDeployGroupToScope < ActiveRecord::Migration
> > +  def change
> > +    remove_index :environment_variables, name: "environment_variables_unique_deploy_group_id"
> > +    add_column :environment_variables, :scope_type, :string
> > +    rename_column :environment_variables, :deploy_group_id, :scope_id
> > +
> > +    EnvironmentVariable.update_all scope_type: "DeployGroup"
>
> some of our older migrations have Model.reset_column_information here,
> whatever works
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/438/files#r31682019>.
>